### PR TITLE
expanded readme

### PR DESCRIPTION
**Pull request by vikiss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,98 +1,113 @@
...
 
-Default name `.sift.yaml` in the root of your project (name can be overriden with an input argument). Set both the API ID (in property `targets[0].api`) and environment ID in the file.  The IDs are obtained from the Aptori Platform.  Refer to documentation about [Sift configuration](https://docs.aptori.dev/sift/configuration/).
+Aptori is an AI-Driven API Security and Application Security Testing platform. With AI-driven generation of functional and security tests, Aptori enables product teams to test, detect and resolve defects during development. Aptori uses an AI-generated semantic graph of your application's API to test the business logic of your application to check for functional defects and all OWASP security vulnerabilities. The Aptori Semantic Tester (sift) integrates with your CI/CD pipelines and tools, such as Jenkins, GitHub, and GitLab.
 
-## Inputs
+## Getting Started:
 
-### `platformKey`
+### 1. Sign up 
+Sign up for a free account with Aptori \
```
Comment (travisnewhouse) [inline README.md:20]: I like that you have added Getting Started guide with steps.  At present time, I don't think we are going to have a free account sign-up.

File: README.md [hunk 2]
```diff
@@ -1,98 +1,113 @@
...
-    - uses: Aptor-dev/sift-action@v1.0.20
-      with:
-        platformKey: ${{ secrets.SIFT_PLATFORM_KEY }}
-        targetUrl: http://vampi:5000
-```
+You need to add `platformKey` and `configurationId` from previous steps to your repository secrets.
 
-### `configurationFile`
+### 6. Run results
+When Sift run completes, the scan results will be available on the [Aptori platform](https://app.aptori.dev)
```
Comment (travisnewhouse) [inline README.md:41]: Is there some way that the action can insert a URL into the Pull Request comments?

File: README.md [hunk 3]
```diff
@@ -1,98 +1,113 @@
...
-        platformKey: ${{ secrets.SIFT_PLATFORM_KEY }}
-        targetUrl: http://vampi:5000
-        configurationFile: .sift/configA.yaml
-```
+## Inputs
 
+### `platformKey`
+**Required** \
+Swrlio platform API key.
+|Default value|`""`|
```
Comment (travisnewhouse) [inline README.md]: All of the default values are blank.  Not adding much.  I think they can be removed.

**Review by travisnewhouse — COMMENTED** [review]
I want to discuss with Sumeet about the trial account.  I have also pushed a commit with some minor edits.

File: README.md (hunk 3 above)
Comment (travisnewhouse) [reply README.md]: I removed them.

File: README.md (hunk 1 above)
Comment (travisnewhouse) [reply README.md]: Removed "free".

File: README.md (hunk 2 above)
Comment (vikiss) [reply README.md]: There is an action for that that we could provide a usage example for. We can also look into publishing the results in SARIF format maybe?..

**Review by travisnewhouse — APPROVED** [review]

**Review by sumeet-aptori — APPROVED** [review]
Looks good!

**Pull request merged**